recipes/ghc/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+ls
```

Comment:
What is this for?

Reply:
I want to ensure that ghc is linked against conda-forge libraries